patternfly/patternfly-elements
# fix: Patch accordion dropshadow

## Component updates

- pfe-accordion


### Related issues

- (#1287) Drop-shadow introduced on accordion heading


### Preview

Link(s) to demo page(s) where this element can be viewed:
- [Link](https://deploy-preview-1288--happy-galileo-ea79c4.netlify.app/elements/pfe-accordion/demo/) 


### What has changed and why

Summarize files edited as part of this MR along with a brief description of what was changed/why.

- A commented line included the styles from the previous line due to a prettier clean-up done incorrectly


### Testing instructions

Be sure to include detailed instructions on how your update can be tested by another developer.

1. Open the demo page for pfe-accordion
2. Click to open the first accordion heading
3. There should be no drop shadow below the heading section


### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
